File: Day-One/Level-Four/python/level_four_test.py
from unittest import TestCase

import level_four

class TestLevelFour(TestCase):

    def test_that_function_returns_array_of_vowel_and_consonant_count(self):

        word = "Hello World"
        expected = [["Vowel", 3],["Consonant", 7]]
        actual = level_four.count_vowels_and_consonant(word)
        self.assertEqual(actual, expected)

    def test_that_get_two_sum_index_returns_the_index_of_the_two_numbers_that_makes_up_the_target(self):

        array = [2, 7, 9, 11, 15]
        target = 9
        actual = level_four.get_two_sum_index(array, target)
        self.assertTrue(actual)

File: Day-One/Level-Four/Javascript/LevelFour.test.js
//LevelFour.test.js
const { get_two_sum_index, count_vowels_and_consonant } = require("./LevelFour");

test(" test that the get_two_sum_index returns a new array of the two numbers that makes up the sum of the target", ()=> {

    let array = [2, 7, 9, 11, 15];
    let target = 9;
    let actual = get_two_sum_index(array, target);
    let expected = [0, 1];

    expect(actual).toEqual(expected);
})

test("ttest  that count_vowels_and_consonant returns the vount of vowel adconsonant in a collection", ()=> {

    let text = "Hello World";
    let actual = count_vowels_and_consonant(text);
    let expected = [["Vowel", 3],["Consonant", 7]]

    expect(actual).toEqual(expected)
})
